.eslintignore

```diff
@@ -0,0 +1,23 @@
+_cdn
+_site
+docs/.jekyll-metadata
+node_modules
+npm-debug.log*
+yarn-error.log*
+.DS_Store
+.history
+yarn.lock
+package-lock.json
+.github
+.storybook
+*.min
+lint.log
+dist
+*.bundle
+scripts
```

Comment:
Why are we including all these files in eslint check? I thought eslint mainly checks for .js files?